diff -up normalize-0.7.7/configure.ac.audiofile normalize-0.7.7/configure.ac
--- normalize-0.7.7/configure.ac.audiofile	2005-09-14 09:23:16.000000000 +0200
+++ normalize-0.7.7/configure.ac	2013-03-25 22:52:11.908477894 +0100
@@ -249,8 +249,8 @@ AC_SUBST(PLUGINS)
 
 dnl *** Stuff for audiofile library ***
 dnl v0.2.1 and before have bugs with 24-bit LE files
-AM_PATH_AUDIOFILE([ 0.2.2 ], [ have_audiofile=true ])
-dnl AC_CHECK_LIB(audiofile, afSetVirtualSampleFormat, have_audiofile=true)
+#AM_PATH_AUDIOFILE([ 0.2.2 ], [ have_audiofile=true ])
+AC_CHECK_LIB(audiofile, afSetVirtualSampleFormat, have_audiofile=true)
 AH_TEMPLATE([USE_AUDIOFILE],
 	    [Define if you want to use the audiofile library.])
 AC_ARG_WITH(audiofile,
@@ -266,7 +266,8 @@ if test x$with_audiofile != xfalse; then
     if test x$have_audiofile = xtrue; then
 	AC_DEFINE(USE_AUDIOFILE)
 	use_audiofile=true
-dnl	AUDIOFILELIBS="-laudiofile"
+	AUDIOFILE_LIBS="-laudiofile"
+	AUDIOFILE_CFLAGS=
     elif test x$with_audiofile = xtrue; then
 	AC_MSG_ERROR([--with-audiofile specified, but audiofile library not found])
     fi
@@ -275,7 +276,8 @@ dnl AC_SUBST(AUDIOFILELIBS)
 test x$use_audiofile = xfalse && AUDIOFILE_LIBS=
 test x$use_audiofile = xfalse && AUDIOFILE_CFLAGS=
 AM_CONDITIONAL(AUDIOFILE, test x$use_audiofile = xtrue)
-
+AC_SUBST(AUDIOFILE_LIBS)
+AC_SUBST(AUDIOFILE_CFLAGS)
 
 dnl *** Stuff for mad mpeg audio decoder library ***
 AH_TEMPLATE([USE_MAD],
